ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -169,3 +169,51 @@ def verification_ui(self):
         for err in self.err_list:
             logger.error(err)
         assert len(self.err_list) == 0, f"{self.err_list}"
+
+    def check_capacity_breakdown(self, project_name, pod_name):
+        """
+        Check Capacity Breakdown
+
+        Args:
+            project_name (str): The name of the project
+            pod_name (str): The name of pod
+
+        Returns:
+            bool: True if project_name and pod_name exist on capacity_breakdown, False otherwise
+
+        """
+        self.navigate_overview_page()
+        if self.ocp_version == "4.7":
```

Comment:
What will happen if OCP version is greater than 4.7?

Reply:
We will not click on `persistent_storage_tab`